docker-compose.yml

```diff
@@ -305,7 +307,7 @@ services:
     profiles:
       - feature-complete
   symbolicator:
-    <<: *restart_policy
+    <<: [*restart_policy, *pull_policy]
```

Comment:
The `*pull_policy` should be removed.

Reply:
I'll remove these from everything that we don't build.